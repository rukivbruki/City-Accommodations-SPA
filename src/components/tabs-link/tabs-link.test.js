import React from "react";
import renderer from "react-test-renderer";
import TabsLink from "./tabs-link";

const city = `Paris`;
const currentCity = `Moscow`;
const handleClick = jest.fn();

it(`Tab-link component render correct`, () => {
  const tabsTemplate = renderer.create(
      <TabsLink
        activeElement={currentCity}
        changeCurrentCity={handleClick}
        city={city}
        id={`o-Paris`}
        onSelect={handleClick}
      />
  ).toJSON();

  expect(tabsTemplate).toMatchSnapshot();
});
