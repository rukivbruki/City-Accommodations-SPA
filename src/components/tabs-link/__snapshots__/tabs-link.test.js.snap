// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tab-link component render correct 1`] = `
<li
  className="locations__item"
>
  <a
    className="locations__item-link tabs__item "
    href="#"
    onClick={[Function]}
  >
    <span>
      Paris
    </span>
  </a>
</li>
`;
